docs/en/observability/apm/tab-widgets/install-agents.asciidoc

```diff
@@ -1,3 +1,79 @@
+// tag::android[]
+*Add the agent to your project*
```

Comment:
By version `1.0` this process will most likely change a lot, so I'm a bit concerned about duplicating it here and possibly forgetting to update it later 😅

Reply:
@bmorelli25 do you think we should remove this until 1.0?

Reply:
That's a fair point, but process changes are inevitable, and I would hate not to have an Android tab on [this page](https://observability-docs_3507.docs-preview.app.elstc.co/guide/en/observability/master/_step_3_install_apm_agents.html).

Is there a 1.0 release issue that we can add a task to so we don't forget to update this?

Reply:
My vote is to move forward with this content as-is and either open an issue or add to an issue to help us not forget to update this content as it changes.

Reply:
Sorry I was OOO, but it sounds good 👍 I created [this issue](https://github.com/elastic/apm-agent-android/issues/260) which, when merged, will make this info no longer accurate.